spring-cloud-netflix-core/src/main/java/org/springframework/cloud/netflix/feign/support/SpringMvcContract.java

```diff
@@ -55,19 +54,24 @@ protected void processAnnotationOnMethod(MethodMetadata data,
 			data.template().method(mapping.method()[0].name());
 
 			// path
-			checkOne(method, mapping.value(), "value");
+			StringBuilder path = new StringBuilder();
```

Comment:
This looks for a @RequestMapping annotation on the class/interface level, and prepends that value (if it exists) to the @RequestMapping annotation's value on the method. @RequestMapping is valid in both locations.